### ISSUE TITLE
OCR - využítí PERO API

### ISSUE THREAD

**Issue by zabak** [LIBCAS/DL4DH#26]
Dokumentace:
https://app.swaggerhub.com/apis-docs/LachubCz/PERO-API/

Link na API, jsou zde i grafy zpracování:
https://pero-ocr.fit.vutbr.cz/api/

Příklady skriptů pro posílání requestů:
https://github.com/DCGM/pero-ocr-api/tree/master/user_scripts

API key zajistím

**Comment by JanMeritus** [comment]
@zabak je zde nejaky posun?

**Comment by zabak** [comment]
Ano, API key pro KNAV jsem dodal Petru Kotýnkovi. @JanMeritus mám dodat i API klíč pro NK?

**Comment by JanMeritus** [comment]
@zabak pokud mozno, to by bylo supr
